R/Aaaa.R

```diff
@@ -93,7 +93,8 @@ Reserved <- c("ID", "amt", "cmt", "ii", "ss", "evid",
               "addl", "rate","time", Reserved_cvar,
               "AMT", "CMT", "II", "SS", "ADDL", "RATE", "THETA",
               paste0("pred_", c("CL", "VC", "V", "V2", "KA", "Q", "VP", "V3")),
-              "double", "int", "bool", "capture")
+              "double", "int", "bool", "capture", 
+              "until", "now")
```

Comment:
Is my understanding correct that 1) `until` is for the functions added in this PR and 2) `now` is for the function added in gh-1149?

Reply:
Right; it took me a while to figure out there would be a clash here (the preprocessor picks up both `ev` and `now` in `ev.now`); I'm wondering about walking this back now that #1159 is in the mix; those should get picked up by this scan.